tests/infer/test_inference.py

```diff
@@ -52,6 +52,9 @@ def test_elbo_reparameterized(self):
     def test_elbo_analytic_kl(self):
         self.do_elbo_test(True, 3000, TraceMeanField_ELBO())
 
+    def test_elbo_tail_adaptive(self):
+        self.do_elbo_test(True, 3000, TraceTailAdaptive_ELBO(num_particles=10, vectorize_particles=True))
```

Comment:
If `TraceTailAdaptive_ELBO` supports nonreparametrized samplers (as your test_valid_models test suggests), then you should add a numerical test that nonreparametrized distributions work. Could you fork this test case?

Reply:
it doesn't support nonreparametrized samplers (as mentioned in docstring). please see test again. unless i'm missing something?